src/ensembl/production/metadata/api/models/genome.py

```diff
@@ -34,7 +34,9 @@ class Genome(LoadAble, Base):
     organism_id = Column(ForeignKey("organism.organism_id"), nullable=False, index=True)
     created = Column(DATETIME(fsp=6), nullable=False)
     is_best = Column(TINYINT(1), nullable=False, default=0)
-    production_name = Column(String(256), nullable=False, unique=False)
+    production_name = Column(String(255), nullable=False, unique=False)
+    genebuild_version = Column(String(20), nullable=True, unique=False)
```

Comment:
The 20 characters for genebuild version may not be enough. Be sure to check with Metazoa as to how they are planning to use it.